math/libfaiss/Portfile

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                libfaiss
+version             20180214
+categories          math
+license             BSD
+platforms           darwin
+maintainers         fb.com:hoss \
+                    fb.com:matthijs
+description         Efficient similarity search library from Facebook AI Research.
+long_description    Library for efficient similarity search and clustering of dense vectors.
```

Comment:
This long description isn't very descriptive (and isn't really longer than the short one.)